scripts/train_crnn.py

```
@@ -0,0 +1,334 @@
+    # 訓練履歴
+    history = {'train_loss': [], 'val_loss': [], 'val_accuracy': []}
+    best_val_accuracy = 0.0
+    
+    # 訓練ループ
+    for epoch in range(epochs):
+        # 訓練
+        model.train()
+        train_loss = 0.0
+        train_batches = 0
+        
+        for images, labels, widths in tqdm(train_loader, desc=f"Epoch {epoch+1}/{epochs} - 訓練"):
```

Ruff (0.12.2)

Loop control variable `widths` not used within loop body

Rename unused `widths` to `_widths`

(B007)